src/cbc_sdk/platform/devices.py

```diff
@@ -589,6 +596,45 @@ def set_target_priorities(self, target_priorities):
         self._update_criteria("target_priority", target_priorities)
         return self
 
+    def set_cloud_provider_account_id(self, cloud_ids):
```

Comment:
Maybe to rename the parameter to be more clear about what it is for - it is a little confusing that the parameter name for both virtual_private_cloud_id and cloud_provider_account_id is called `cloud_ids`.

Reply:
I changed that parameter name.

Reply:
Thanks.